src/MidiPlayer.js

```diff
@@ -624,4 +625,36 @@ not.
         this.logging = logging;
         this.eventHandler.setLogger({ eventLogger, logging });
     }
+    /*
+     * get total duration time
+     * 2021 03 NormalUniversity
```

Comment:
Please refrain from adding some sort of authorship comments in the code. This is not a sustainable method to get credit for your contribution. If you want to be recognized for your contribution, you are welcome to add your contact information in the `package.json` file in the `contributors` array as documented here: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors.